databrowser/src/components/pill/PillGroup.vue

```diff
@@ -1,44 +1,24 @@
 <template>
   <div>
-    <div
+    <PillGroupItem
```

Comment:
What do you think about renaming this component to `PillButtonGroup` and `PillButtonGroupItem`? So it is more consistent with the link variant.